src/base/rss/rss_feed.cpp

```diff
@@ -50,21 +51,30 @@
 #include "rss_article.h"
 #include "rss_session.h"
 
-const QString Str_Url(QStringLiteral("url"));
-const QString Str_Title(QStringLiteral("title"));
-const QString Str_LastBuildDate(QStringLiteral("lastBuildDate"));
-const QString Str_IsLoading(QStringLiteral("isLoading"));
-const QString Str_HasError(QStringLiteral("hasError"));
-const QString Str_Articles(QStringLiteral("articles"));
+const QString KEY_UID(QStringLiteral("uid"));
+const QString KEY_URL(QStringLiteral("url"));
+const QString KEY_TITLE(QStringLiteral("title"));
+const QString KEY_LASTBUILDDATE(QStringLiteral("lastBuildDate"));
+const QString KEY_ISLOADING(QStringLiteral("isLoading"));
+const QString KEY_HASERROR(QStringLiteral("hasError"));
+const QString KEY_ARTICLES(QStringLiteral("articles"));
 
 using namespace RSS;
 
-Feed::Feed(const QString &url, const QString &path, Session *session)
+Feed::Feed(const QUuid &uid, const QString &url, const QString &path, Session *session)
     : Item(path)
     , m_session(session)
+    , m_uid(uid)
     , m_url(url)
 {
-    m_dataFileName = QString("%1.json").arg(Utils::Fs::toValidFileSystemName(m_url, false, QLatin1String("_")));
+    m_dataFileName = (QString {"%1.json"}).arg(QString::fromLatin1(m_uid.toRfc4122().toHex()));
+
+    // Move to new file naming scheme (since v4.1.2)
+    const QString legacyFilename {
+        QString("%1.json").arg(Utils::Fs::toValidFileSystemName(m_url, false, QLatin1String("_")))};
```

Comment:
I would use `+` to concate the strings as QStringBuilder can optimize it a bit.

Reply:
I prefer using formatted strings to clearly see the resulting structure. But in case of only two parts I don't object to use `+` to concatenate it.